week8/src/Main.java

```diff
@@ -0,0 +1,183 @@
+            System.out.println("[3] Retrieve all students by age");
+            System.out.println("[4] Exit");
+
+            int tempChoice = sc.nextInt();
+            choice = tempChoice;
+
+            switch (choice) {
+
+                //Add student
+                case 1:
+                    String name = name();
+                    String surname = surname();
+                    int year = birthYear();
+                    String gender = genderSelect();
+                    int id = generateId();
+                    sList.add(new Student(name,surname,year,gender,id));
+                    System.out.println("Student successfully added! Press [Enter] to continue...");
+                    try{System.in.read();}
+                    catch(Exception e){}
+                    break;
+
+                //Remove student by ID
+                case 2:
+                    sList.forEach(System.out::println);
+                    int x = removeStudent();
+                    for (int i = 0; i < sList.size(); i++) {
+                        if (sList.get(i).getId() == x){
+                            sList.remove(i);
+                            System.out.println("Student successfully removed! Press [Enter] to continue...");
+                        }
+                    }
+                    try{System.in.read();}
+                    catch(Exception e){}
+                    break;
+
+                //Retrieve students by age
+                case 3:
+                    System.out.println("Enter age: ");
+                    int age = sc.nextInt();
+                    int birthYear = 2019 - age;
+
+                    for (int i = 0; i < sList.size(); i++) {
+                        if (sList.get(i).getBirthYear() == birthYear){
+                            byAge.add(sList.get(i));
+                        }
+                    }
+
+                    if (byAge.isEmpty()) {
+                        System.out.println("There are no " + age + " years old students.");
+                    } else {
+                        byAge.forEach(System.out::println);
+                    }
+                    byAge.clear();
+                    System.out.println("Press [Enter] to continue...");
+                    try{System.in.read();}
+                    catch(Exception e){}
+                    break;
+
+                //Exit program
+                case 4:
+                    System.out.println("Exiting Program...");
+                    System.exit(0);
+                    break;
+
+                default:
+                    System.out.println("Error! Wrong Input");
+                    break;
+            }
+
+        } while (choice != 4);
+
+    }
+
+
+    private static int removeStudent(){
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Enter student ID to remove him..");
+        int id = sc.nextInt();
+        return id;
+    }
+
+    private static String name() throws IOException {
```

Comment:
As a rule, methods' names should be constructed using verbs; something like `getName()`